src/multipliers/NFTMultiplier.sol

```diff
@@ -60,10 +60,10 @@ contract NFTMultiplier is INFTMultiplier, Initializable, Ownable2StepUpgradeable
         return validUntilBlock;
     }
 
-    /// @notice Update the multiplying factor
-    /// @param _newMultiplyingFactor New multiplying factor (in basis points)
-    function updateMultiplyingFactor(uint256 _newMultiplyingFactor) external onlyOwner {
-        multiplyingFactor = _newMultiplyingFactor;
+    /// @notice Updates the multiplying factor for a specific user
+    /// @param _user The address of the user to update the multiplying factor for
+    function updateMultiplyingFactor(address _user) external view onlyOwner {
```

Comment:
Should remove the `onlyOwner` modifier here. Otherwise this will revert if anything but the owner calls it. The "YieldDistributor" contract likely won't be the owner.